new-frontiers-14/frontier-station-14
# Fix time reference in dead drop hints

## About the PR



Makes the dead drop hints use the same time reference as the PDA clock (no more 17 hour hints at round start).

Reduces redundancy - the random smuggling fax event uses a function in the dead drop system.

## Why / Balance


Time reference was a bug, addresses this: https://discord.com/channels/1123826877245694004/1290779879024885761

Redundancy: making my own code a bit less awful.

## How to test


1. Start the server, check the server console output to find dead drop hints, check the time reference in the dead drop hints.  Should be fine.
2. Purchase a ship, run the SmugglingFax event (`addgamerule SmugglingFax`), check the time reference in the dead drop hints, it should be fine.

## Media


## Requirements



## Breaking changes


**Changelog**


:cl:
- fix: The time printed out for dead drop hints is now accurate to shift time.
